pkgs/development/compilers/avian/default.nix

```diff
@@ -0,0 +1,35 @@
+{ stdenv, fetchFromGitHub, zlib, jdk }:
+
+stdenv.mkDerivation {
+  name = "avian-1.2.0";
+
+  src = fetchFromGitHub {
+    owner = "readytalk";
+    repo = "avian";
+    rev = "72c0547f4bdedc1ba8da2882317e3f758deab23f";
```

Comment:
I like tags because they're more convenient. While tags *are* mutable, overwriting release tags is a terrible practice and I prefer to pretend it doesn't happen. Also, consider that all the URLs we use throughout Nixpkgs are mutable (in the sense that nothing prevents upstream from replacing an archive with another of the same name) so we're no worse off for using release tags here.